Gemfile

```diff
@@ -30,6 +30,7 @@ gem "sentry-rails"
 gem "twitter-bootstrap-rails"
 gem "twitter-bootstrap-rails-confirm"
 gem "uglifier"
+gem "webrick"
```

Comment:
lol wut

Reply:
oooo you know what, look at this: https://github.com/artsy/doppler/blob/6a13a1cd7fa52026fe5c983be2134c26cfd52282/spec/support/capybara.rb#L5

Reply:
lets kill that and then remove the need for `webrick` altogether!

Reply:
Also found some newrelic config in here! Removing